source/game.programme.programmelicence.bmx

```diff
@@ -3081,7 +3081,11 @@ Type TProgrammeLicenceFilter
 	Method DoesFilter:Int(licence:TProgrammeLicence)
 		if not licence then return False
 
-		if checkAvailability and not licence.isAvailable() then return False
+		'if a licence is exceeding the broadcast limit and it is still owned by the player
```

Comment:
Ich denke der Kommentar ist hier "missverstaendlich".

Entweder muesste man wirklich noch den "owner" ueberpruefen und nur bei "Spielerbesitz" die "availability" pruefen - oder eben nicht.

Beim Haendler wuerden die Limits ja - wenn erlaubt - wieder aufgefuellt worden sein. Wenn nicht, entfernt "GiveBackToLicencePool" ja eh den Tradeable-Flag (die folgende Ueberpruefung).
Ich wuerde also sagen, dass dieses "and it is still owned by the player" weg kann.